receiver/splunkhecreceiver/receiver_test.go

```diff
@@ -456,7 +456,7 @@ func Test_splunkhecReceiver_TLS(t *testing.T) {
 		require.NoError(t, r.Shutdown(context.Background()))
 	}()
 
-	mh := newAssertNoErrorHost(t)
+	mh := newAssertNoErrHost(t)
```

Comment:
Unrelated changes?

Reply:
this is related, there is a name collision on `newAssertNoErrorHost` otherwise for this component.